pandas/core/internals/blocks.py

```diff
@@ -431,7 +433,9 @@ def _maybe_downcast(self, blocks: list[Block], downcast=None) -> list[Block]:
         if downcast is None:
             return blocks
 
-        return extend_blocks([b._downcast_2d(downcast) for b in blocks])
+        return extend_blocks(
+            [b._downcast_2d(downcast) for b in blocks]  # type: ignore[call-arg]
```

Comment:
Did something change here, or also a merge leftover?

Reply:
Same yes, Updated everything now, should be good to go.